verifact/forum/schema.py

```diff
@@ -118,7 +126,53 @@ def mutate_and_get_payload(
 
         return AnswerCreate(answer=answer)
 
+class VoteUpdate(ClientIDMutation):
+    vote = Field(VoteNode)
+
+    class Input:
+        credible = Boolean() # if null, remove vote!
```

Comment:
I think this should be a required field, i.e. `credible = Boolean(required=True)`.

Reply:
currently the implementation recognises that the user wants to delete a Vote by choosing to not supply a `credible` boolean - do you have any suggestions on a cleaner way to do this?